.travis.yml

```diff
@@ -22,31 +22,17 @@ cache:
 
 env:
   matrix:
-  - EMBER_TRY_SCENARIO=ember-1-12
```

Comment:
No, it can be removed. I have this listed as a task in the dev ops project to do across all the repos.